docs/overview.md

```diff
@@ -19,72 +19,19 @@ The basic idea is this:
 People are giving of themselves and their free time to contribute to open source projects in so many ways. It can be a real
 time sink sometimes and so they should be praised for all their contributions (code or not).
 
-Use this project as an example implementation of the all-contributors specification.
+*TODO: Insert screenshot*
 
-## Tools for updating your all-contributors table
-
-If you find maintaining the list of contributors tedious, say hello to the [@all-contributors bot 🤖](https://github.com/all-contributors/all-contributors-bot)
-
-If you prefer command line tools, there is also the [all-contributors CLI tool](https://www.npmjs.com/package/all-contributors-cli)
-
-## Specification
-
-### Mandatory
-Open source projects should include the following mandatory items in order to support the All Contributors specification:
+## From here?
+    - The all contributors specification
```

Comment:
Shouldn't that be a link to `docs/specification.md` since that part it covers was initially on that file?